spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaListenerAnnotationBeanPostProcessor.java

```diff
@@ -563,18 +564,16 @@ protected void processListener(MethodKafkaListenerEndpoint<?, ?> endpoint, Kafka
 		processKafkaListenerAnnotationBeforeRegistration(endpoint, kafkaListener, bean, topics, tps);
 
 		String containerFactory = resolve(kafkaListener.containerFactory());
-		KafkaListenerContainerFactory<?> listenerContainerFactory = resolveContainerFactory(kafkaListener, containerFactory, beanName);
+		KafkaListenerContainerFactory<?> listenerContainerFactory = resolveContainerFactory(kafkaListener,
+				containerFactory, beanName);
 
 		this.registrar.registerEndpoint(endpoint, listenerContainerFactory);
-
-		processKafkaListenerEndpointAfterRegistration(endpoint, kafkaListener);
 	}
 
 	private void processKafkaListenerAnnotationForRetryTopic(MethodKafkaListenerEndpoint<?, ?> endpoint,
```

Comment:
Maybe we can lose this method and take off the "beforeRegistration" part of the other, since now there's no "after"?